Os/Baremetal/TaskRunner/BareTaskHandle.hpp

```diff
@@ -19,7 +19,7 @@ class BareTaskHandle {
         bool m_enabled;
         //!< Save the priority
         NATIVE_INT_TYPE m_priority;
-        //!< Function passed in to the task
+        //!< Function passed into the task
         Task::taskRoutine m_routine;
         //!< Argument input pointer
         void* m_argument;
```

Comment:
I can't hang a comment on renaming files, but the next two are instances of correcting `Generic`. Oddly, there don't appear to be any instances of `Genric` in the code...

Reply:
Those files are used to visualize on behalf of the documentation and aren't directly instantiated.